vtex/faststore
# Home: migrating product shelf section to app folder

## What's the purpose of this pull request?

Migration of the ProductShelf section from the pages router to the app router (Next 13).

## How it works?

<img width="1512" alt="Captura de Tela 2024-08-15 às 10 27 30" src="https://github.com/user-attachments/assets/d70f674c-a634-47a2-b03c-222e94147b86">


## How to test it?

How to test it?
Run `yarn dev` and inside `packages/core` run `yarn build && yarn serve` 
Make sure that all the current pages are working as expected (/, /office and some /<slug>/p);
Make sure /fs-next-update is rendering with the ProductShelf section without errors.
